tests/integration/cluster-users/sar-test.sh

```diff
@@ -240,6 +255,94 @@ function runTests() {
 
 	echo "--> ${VIEW_EMAIL} should NOT be able to create ory Access Rule"
 	testPermissions "create" "rule.oathkeeper.ory.sh" "${NAMESPACE}" "no"
+
+	EMAIL=${NAMESPACE_ADMIN_EMAIL} PASSWORD=${NAMESPACE_ADMIN_PASSWORD} getConfigFile
+	export KUBECONFIG="${PWD}/kubeconfig"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to create new namespace"
+	createNamespaceForNamespaceAdmin
+	export SHOULD_CLEANUP_NAMESPACE="true"
+
+	# namespace admin should not be able to get or create any resource in system namespaces
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to list Deployments in system namespace"
+	testPermissions "list" "deployment" "${SYSTEM_NAMESPACE}" "no"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to get ory Access Rule in system namespace"
+	testPermissions "get" "rule.oathkeeper.ory.sh" "${SYSTEM_NAMESPACE}" "no"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create secret in system namespace"
+	testPermissions "create" "secret" "${SYSTEM_NAMESPACE}" "no"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to delete system namespace"
+	testPermissions "delete" "namespace" "${SYSTEM_NAMESPACE}" "no"
+
+	# namespace admin should not be able to create clusterrolebindings - if they can't create it in one namespace, 
+	# that means they can't create it in any namespace (resource is non namespaced and RBAC is permissive)
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to create clusterrolebindings"
+	testPermissions "create" "clusterrolebinding" "${SYSTEM_NAMESPACE}" "no"
+
+	# namespace admin should be able to get/create k8s and kyma resources in the namespace they created
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to list Deployments in the namespace they created"
+	testPermissions "list" "deployment" "${CUSTOM_NAMESPACE}" "yes"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to get Pods in the namespace they created"
+	testPermissions "get" "pod" "${CUSTOM_NAMESPACE}" "yes"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to create ory Access Rule in the namespace they created"
+	testPermissions "create" "rule.oathkeeper.ory.sh" "${CUSTOM_NAMESPACE}" "yes"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to create secret in the namespace they created"
+	testPermissions "create" "secret" "${CUSTOM_NAMESPACE}" "yes"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to delete namespace they created"
+	testPermissions "delete" "namespace" "${CUSTOM_NAMESPACE}" "yes"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to create rolebindings to kyma-developer clusterrole in the namespace they created"
+	createRoleBindingForNamespaceDeveloper
```

Comment:
Please add check for :  NS ADMIN create clusterrolebinding, expected output : NO

Reply:
See line 281: https://github.com/kyma-project/kyma/pull/6892/files#diff-2d6db4580a2bbed5c105df02c9cad3daR281
and this conversation: https://github.com/kyma-project/kyma/pull/6892#discussion_r370048643